packages/shared-ux/button_toolbar/src/toolbar/toolbar.test.tsx

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import { mountWithIntl } from '@kbn/test-jest-helpers';
+
+import { Toolbar } from './toolbar';
+import { PrimaryButton } from '../buttons';
+
+describe('<Toolbar />', () => {
+  test('is rendered', () => {
+    const primaryButton = <PrimaryButton label="Create chart" onClick={() => 'click'} />;
```

Comment:
nit: `onClick` be just empty function if we're not testing if it can be clicked. but can we add a test that click actually calls the function?

Reply:
Good call - [379590c](https://github.com/elastic/kibana/pull/137224/commits/379590c375b2a32c4d524db8d309f6549bae196f) I got a little goofed up trying to get only the button clicked and I think this test nails it thanks!